ClientSupport/src/main/java/io/deephaven/clientsupport/gotorow/SeekRow.java

```diff
@@ -45,155 +42,147 @@ public SeekRow(long startingRow, String columnName, Object seekValue, boolean in
+                        row = findRow(rowSet, (int) startingRow + 1, (int) rowSet.size());
+                        if (row >= 0) {
+                            result.setValue(row);
+                            log.info().append("found 3").endl();
+                            return true;
+                        }
+                        row = findRow(rowSet, 0, (int) startingRow + 1);
+                        if (row >= 0) {
+                            result.setValue(row);
+                            log.info().append("found 4").endl();
+                            return true;
+                        }
+                    }
+                    result.setValue(-1L);
+                    return true;
+                }));
 
-        // just go to the closest value
-        if (closestLowerValueYet == null && closestUpperValueYet == null) {
-            return -1L;
-        } else if (closestLowerValueYet == null) {
-            return index.find(closestUpperRowYet);
-        } else if (closestUpperValueYet == null) {
-            return index.find(closestLowerRowYet);
-        } else {
-            // we need to decide between the two
-            Class columnType = table.getColumnSource(columnName).getType();
-            if (Number.class.isAssignableFrom(columnType)) {
-                double nu = ((Number) closestUpperValueYet).doubleValue();
-                double nl = ((Number) closestLowerRowYet).doubleValue();
-                double ns = ((Number) seekValue).doubleValue();
-                double du = Math.abs(nu - ns);
-                double dl = Math.abs(nl - ns);
-                log.info().append("Using numerical distance (").appendDouble(dl).append(", ").appendDouble(du)
-                        .append(")").endl();
-                return index.find(du < dl ? closestUpperRowYet : closestLowerRowYet);
-            } else if (Instant.class.isAssignableFrom(columnType)) {
-                long nu = DateTimeUtils.epochNanos(((Instant) closestUpperValueYet));
-                long nl = DateTimeUtils.epochNanos(((Instant) closestLowerValueYet));
-                long ns = DateTimeUtils.epochNanos(((Instant) seekValue));
-                long du = Math.abs(nu - ns);
-                long dl = Math.abs(nl - ns);
-                log.info().append("Using nano distance (").append(dl).append(", ").append(du).append(")").endl();
-                return index.find(du < dl ? closestUpperRowYet : closestLowerRowYet);
-            } else {
-                long nu = index.find(closestUpperRowYet);
-                long nl = index.find(closestLowerRowYet);
-                long ns = startingRow;
-                long du = Math.abs(nu - ns);
-                long dl = Math.abs(nl - ns);
-                log.info().append("Using index distance (").append(dl).append(", ").append(du).append(")").endl();
-                return du < dl ? nu : nl;
-            }
-        }
+        log.info().append(result.getValue()).endl();
```

Comment:
Instead of logging this, just a null check so we can alert that something didn't work?
```suggestion
        Assert.neqNull(result.getValue(), "result.getValue()");
```